src/styles/Token.scss

```diff
@@ -1,5 +1,5 @@
 .psyoptions-token {
   .p-content img {
-    width: 100%;
+    // width: 100%;
```

Comment:
nit: should remove if we're going to leave this commented

Reply:
> How come the package-lock.json is deleted?

What do you mean?

Reply:
@haroonpsyoptions if a comment does not add any value to another dev in the codebase then it should not exist. I have removed the comment and the unnecessary scss file